Setup/ericomshield-setup.sh

```diff
@@ -515,12 +511,22 @@ function wait_for_docker_to_settle() {
     done
 }
 
+function set_storage_driver() {
+    if [ -f /etc/docker/daemon.json ] && [ $(grep -c '"storage-driver"[[:space:]]*:[[:space:]]*"overlay2"' /etc/docker/daemon.json) -eq 1 ]; then
+        echo '"storage-driver": "overlay2" in /etc/docker/daemon.json'
+    else
+        systemctl stop docker && \
+        cat /etc/docker/daemon.json | jq '. + {storage-driver: "overlay2"}' >/etc/docker/daemon.json.shield && \
+        echo 'Setting: "storage-driver": overlay2 in /etc/docker/daemon.json' && \
+        mv /etc/docker/daemon.json.shield /etc/docker/daemon.json && \
+        systemctl start docker || exit 1
```

Comment:
I think we also need to remove (or move) the contents of the existing /var/lib/docker before restarting Docker.

Reply:
note that for now, we are doing overlay2 only for new installation
I prefer to not do it for updates, since i understand we will lose data,
so I'll recommend to do it manually (only if required like in prod-Jer)
Do we still need to remove/move?
if yest please do the change

Also, please check the error:
./ericomshield-setup.sh: line 327: /etc/dnsmasq.d/ericom-shield: No such file or directory

Reply:
The error should be fixed now.